app/assets/stylesheets/govuk_publishing_components/components/_previous-and-next-navigation.scss

```diff
@@ -1,6 +1,6 @@
 .gem-c-pagination {
   display: block;
-  margin: govuk-spacing(6) (- govuk-spacing(3) );
+  margin: govuk-spacing(7) (0);
```

Comment:
Have you tested this change in an application? I've got a feeling it has this slightly odd negative margin for a reason.

Reply:
Yes tested on an application. It would seem the original (- govuk-spacing(3)) extends the pagination area to the edge of device where the padding of the link then brings it back in-line. Not sure on the reasoning. 

Updated code puts the padding on the link only as the pagination area has no left or right margin to its parent so is inline